src/client/entity-editor/entity-editor.tsx

```diff
@@ -80,11 +84,17 @@ const EntityEditor = (props: Props) => {
 		window.onbeforeunload = handleUrlChange;
 	}, [handleUrlChange]);
 
+	if(entity){
+	entityURL = getEntityUrl(entity);
+	}
+	
 	return (
 		<form onSubmit={onSubmit}>
 			<Card>
 				<Card.Header as="h4">
-					{heading}
+					<div>
+					 <a href={entityURL}>{heading}</a>
+					</div>
```

Comment:
Following my comment above, in the case that entity is falsy we know it's not an edit but a creation, in which case we can use the `heading` directly instead of creating a broken link.
`{entityURL ? <a href={entityURL}>{heading}</a> : heading}`

Reply:
oh nice , I think that will be much easier to understand for others too !

Reply:
yup I think this is much better